sdv/constraints/base.py

```diff
@@ -169,16 +127,21 @@ def _validate_data_on_constraint(self, table_data):
 
                 raise ConstraintsNotMetError(err_msg)
 
-    def check_missing_columns(self, table_data):
-        """Check ``table_data`` for missing columns.
+    def _fit(self, table_data):
+        del table_data
```

Comment:
I'm wondering if we need this. Couldn't we just use `pass` or I'm missing something ?

Reply:
we could, this is just from before